src/API/Router.ts

```diff
@@ -82,8 +82,7 @@ export class Router {
   }
 
   private getWorks = async (context: KoaRouter.IRouterContext, next: () => Promise<any>) => {
-    this.logger.trace({ query: context.query }, 'GET /works')
-
+    this.logger.trace({ query: context.query }, '/works?publicKey=')
     const publicKey = context.query.publicKey
     const works = await this.workController.getByPublicKey(publicKey)
```

Comment:
If we implement the  `getByFilters` as I mention in my other comment and if Joi validates that no unknown query parameters are being passed, we could just `const works = await this.workController.getByPublicKey(context.query)` here.

@wzalazar do you know if Joi rejects unexpected parameters?

Reply:
Joi reject all request that contains unknown parameter unless you has enabled the option to force accept it parameters

Reply:
Thanks @wzalazar! Sounds like we can `const works = await this.workController.getByPublicKey(context.query)` :smile: